frontend/src/components/new-car-card/new-car-card.tsx

```diff
@@ -40,7 +30,14 @@ const NewCarCard: React.FC<ExtendedCarCardPropsType> = (props) => {
     },
   } = props;
 
-  const [isLiked, setIsLiked] = useState(Boolean(wishlistId));
+  const [isHidden, setIsHidden] = useState<boolean>(true);
```

Comment:
What does it hide?

Reply:
We hide `CompareToast` component (notification). Please look at https://github.com/BinaryStudioAcademy/bsa-2022-autoline/pull/196#discussion_r962349299